src/main/resources/application.conf

```diff
@@ -0,0 +1,25 @@
+# locations
+bucket_name = "mybucket"
+bucket_landing_path = /landing
+bucket_staging_path = /staging
```

Comment:
I was planning to have searches to hit both staging and landing. 

The staging will contain Parquet files with optimised Parquet files (written by the Merger), while landing will potentially have multiple ones. Having search hitting just /staging, would mean events that have arrived but not been merged, do not show.